src/drivers/boards/aerocore/CMakeLists.txt

```diff
@@ -37,10 +37,10 @@ px4_add_module(
 	SRCS
 		../common/board_name.c
 		aerocore_init.c
-		aerocore_pwm_servo.c
+		aerocore_timer_config.c
```

Comment:
Accidentally committed that as I noticed it while changing the pwm configuration for the CF2. It should be noted that aerocore won't build as pwm_servo.c no longer exists and has been renamed to timer_config.c